### PR TITLE
Experiment HMEM

### PR DESCRIPTION
**Pull request by Damenus**

### CODE REVIEW

File: examples/kubernetes/monitoring/wca/wca-config.yaml [hunk 1]
```diff
@@ -24,23 +24,34 @@ runner: !MeasurementRunner
 
   extra_labels:
     node: !Env HOSTNAME
-  event_names:
-    - task_cycles # do not remove: USED BY E2E tests
-    - task_instructions
-    - task_offcore_requests_demand_data_rd # required fro R/W characterization
-    - task_offcore_requests_demand_rfo
+#  event_names:
```
Comment (mmucek95) [inline examples/kubernetes/monitoring/wca/wca-config.yaml:27]: This should be commented out for experiment purposes but please do not commit this to the repo.

File: examples/kubernetes/monitoring/wca/wca-config.yaml [hunk 2]
```diff
@@ -24,23 +24,34 @@ runner: !MeasurementRunner
...
+#    - task_instructions
+#    - task_offcore_requests_demand_data_rd # required fro R/W characterization
+#    - task_offcore_requests_demand_rfo
   enable_derived_metrics: True
   uncore_event_names:
-    - platform_cas_count_reads
-    - platform_cas_count_writes
-    - platform_pmm_bandwidth_reads
-    - platform_pmm_bandwidth_writes
+    # grupa 1
```
Comment (mmucek95) [inline examples/kubernetes/monitoring/wca/wca-config.yaml:34]: Please write in english.

File: examples/kubernetes/scripts/analyzer/metrics.py [hunk 3]
```diff
@@ -74,6 +87,18 @@ class Metric(Enum):
     Metric.PLATFORM_RPQ_READ_LATENCY_SECONDS,
 ]
 
+migration_platform_metrics = [
+    # Metric.PLATFORM_VMSTAT_NUMA_HINT_FAULTS,
```
Comment (mmucek95) [inline examples/kubernetes/scripts/analyzer/metrics.py:91]: Please remove unused

File: examples/kubernetes/scripts/analyzer/metrics.py [hunk 4]
```diff
@@ -154,6 +225,46 @@ class Metric(Enum):
...
+    Metric.PLATFORM_VMSTAT_NUMA_HINT_FAULTS:
+        'platform_vmstat_numa_hint_faults',
+    Metric.PLATFORM_VMSTAT_NUMA_HINT_FAULTS_LOCAL:
+        'platform_vmstat_numa_hint_faults_local',
+    Metric.PLATFORM_VMSTAT_PGMIGRATE_SUCCESS:
+        'platform_vmstat_pgmigrate_success',
+    Metric.PLATFORM_VMSTAT_PGMIGRATE_FAIL:
+        'platform_vmstat_pgmigrate_fail',
+
+    # free vs toptier watermark (when free(solid) below toptier mark wakeup kswapd/toptier_reclam)
```
Comment (mmucek95) [inline examples/kubernetes/scripts/analyzer/metrics.py:238]: Please remove unused code instead of commenting it out.

**Review by mmucek95 — CHANGES_REQUESTED** [review]

File: examples/kubernetes/monitoring/wca/wca-config.yaml (hunk 1 above)
Comment (Damenus) [reply examples/kubernetes/monitoring/wca/wca-config.yaml]: I reverted the changes

File: examples/kubernetes/scripts/analyzer/metrics.py (hunk 3 above)
Comment (Damenus) [reply examples/kubernetes/scripts/analyzer/metrics.py:91]: as above

File: examples/kubernetes/monitoring/wca/wca-config.yaml (hunk 2 above)
Comment (Damenus) [reply examples/kubernetes/monitoring/wca/wca-config.yaml]: as above

File: examples/kubernetes/scripts/analyzer/metrics.py (hunk 3 above)
Comment (Damenus) [reply examples/kubernetes/scripts/analyzer/metrics.py:91]: Deleted

**Review by mmucek95 — APPROVED** [review]

**Review by felidadae — APPROVED** [review]

**Pull request merged**